lib/factory_girl/linter.rb

```diff
@@ -1,11 +1,12 @@
 module FactoryGirl
   class Linter
-    def self.lint!(factories_to_lint)
-      new(factories_to_lint).lint!
+    def self.lint!(factories_to_lint, options)
```

Comment:
Thanks for the input. I think I see what you're saying. Is it mainly one or both?
* you'd like an API that separates the single lint! call into two calls? I naturally wrote ``lint!(factories, traits: true)`-- a single-- but can certainly appreciate other opinions
* `calculate_invalid_factories` is too complex, doing two different things.

Let me know, and I can rev per your feedback. I'm eager to get this merged.